### DISCUSSION TITLE
Is there any language specification?

### DISCUSSION THREAD

**Discussion by IWANABETHATGUY** [less/less.js#3710] [Q&A] [answered]
I want to write a simple parser use tree-sitter, but i could't found any spec neither website and repo doc, so i am curious if the spec is exist somewhere

**Comment by iChenLei** [comment, marked as answer]
Sorry, There isn't less language spec for you, but you can learn something from vscode-css-languageservice [less part](https://github.com/microsoft/vscode-css-languageservice/blob/main/src/parser/lessParser.ts) or [postcss-less](https://github.com/shellscape/postcss-less/blob/master/lib/LessParser.js).
